pkg/investigations/aiassisted/aiassisted.go

```
@@ -37,7 +39,7 @@ type InvestigationPayload struct {
 func generateSessionID(incidentID string) string {
 	timestamp := time.Now().Unix()
 	randomBytes := make([]byte, 8)
-	rand.Read(randomBytes) //nolint:errcheck,gosec
+	rand.Read(randomBytes) //nolint:errcheck
```

golangci-lint (2.11.4)

[error] directive `//nolint:errcheck` is unused for linter "errcheck"

(nolintlint)